src/wazuh_modules/wm_syscheck.c

```diff
@@ -0,0 +1,271 @@
+/*
+ * Wazuh SYSCHECK
+ * Copyright (C) 2015-2021, Wazuh Inc.
+ * March 31, 2021.
+ *
+ * This program is free software; you can redistribute it
+ * and/or modify it under the terms of the GNU General Public
+ * License (version 2) as published by the FSF - Free Software
+ * Foundation.
+ */
+#include <stdlib.h>
+#include "../../wmodules_def.h"
+#include "wmodules.h"
+#include "wm_syscheck.h"
+#include "syscheckd/syscheck.h"
+#include "rootcheck/rootcheck.h"
+#include "sym_load.h"
+#include "defs.h"
+#include "mq_op.h"
+
+static void* wm_syscheck_main(wm_syscheck_t *sys);        // Module main function. It won't return
+static void wm_syscheck_destroy(wm_syscheck_t *sys);      // Destroy data
+static cJSON *wm_syscheck_dump(const wm_syscheck_t *sys);
+static int wm_syscheck_sync_message(const char *data){
```

Comment:
This is not needed. You can just remove this and, in line 34, add `NULL` instead